src/win32/draw.c

```diff
@@ -432,7 +433,7 @@ struct image {
 #define Height(i) (((struct image *)Data_custom_val(i))->h)
 #define Data(i) (((struct image *)Data_custom_val(i))->data)
 #define Mask(i) (((struct image *)Data_custom_val(i))->mask)
-#define Max_image_mem 500000
+#define Max_image_mem 2000000
```

Comment:
Why this change?

Reply:
Reason is in the commit msg: "Synchronize Max_image_mem between unix and win32 backends"

Reply:
Got it